pkg/tsdb/cloudwatch/models/cloudwatch_query_test.go

```diff
@@ -11,10 +11,12 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	"github.com/grafana/grafana/pkg/infra/log/logtest"
+	log "github.com/grafana/grafana/pkg/infra/log"
 	"github.com/grafana/grafana/pkg/tsdb/cloudwatch/utils"
 )
 
+var logger = log.New("testCloudWatchLogger")
```

Comment:
Is there a reason that we're using an actual logger instead of `&logtest.Fake{}`? Does this logger print out to anywhere?

Reply:
No reason, just copied from another test file. I'll use `&logtest.Fake{}` instead.